### ISSUE TITLE
Update form config subtitle option

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#87470]
### Description

The `subTitle` within the form config now accepts passing in a function.

| Options | Value Type | Description | Example |
|---|---|---|---|
| subTitle | string or function | The subtitle of the form, usually the form number. The subtitle is rendered on all pages when there's also a title. Accepts a string or function; params include (currentLocation, formConfig, formData, children). | `subTitle: 'VA Form 10182 (Notice of Disagreement)'` or `subTitle: ({ formData }) => formData.changeSubTitle ? 'Form 345 with Form 123' : 'Form 123'` |

### Relevant URLs

- Doc page: https://depo-platform-documentation.scrollhelp.site/developer-docs/va-forms-library-form-config-options
- Pull request associated with the work: https://github.com/department-of-veterans-affairs/vets-website/pull/30206

### Which type of team are you on? (Platform team, VFS team, or Leadership)

VFS team: Benefits Decision Reviews

**Comment by jknipes** [comment]
Hi @Mottie , thanks for your content suggestions. The Platform Content team will review your request and put it in our backlog to refine for next steps.

**Comment by jknipes** [comment]
@Mottie could you please advise how the suggested change would look on Platform Website > va forms library form config options page?
It appears the suggestion is a simple update to that text on the page

**Comment by Mottie** [comment]
Yes, it's an update to the text already on the page. The `subTitle` option now supports passing in a function. Previously, it only supported a text value.

**Comment by jknipes** [comment]
@taylojill would you be able to check and see if this update is done? Likely we can close this ticket.

**Comment by jknipes** [comment]
Issue completed in [87579](https://github.com/orgs/department-of-veterans-affairs/projects/1463/views/1?filterQuery=87470&pane=issue&itemId=90061034&issue=department-of-veterans-affairs%7Cva.gov-team%7C87579)
